qiskit_experiments/framework/backend_data.py

```diff
@@ -205,29 +205,6 @@ def provider(self):
             return None
         return None
 
-    @property
-    def drive_freqs(self):
```

Comment:
I think we don't need to delete these methods. `meas_freqs` has note about missing frequency, and I think this is sufficient to communicate with users. Actually I don't like V1 model, because we have qubit frequencies in three places, i.e. `backend.defaults().qubit_freq_est`, `backend.properties().qubit_properties()`, and `backend.configuration().hamiltonian`. This method unifies where to get frequency, and good for novice developers.

Probably we can update API so that this can take index of target element -- like t1 method you implemented.

Reply:
One thing that confuses me is the role of `PulseDefaults` now. I opened https://github.com/Qiskit/qiskit-experiments/issues/877 about this. `BackendData` provides compatibility between V1 and V2, but `defaults()` is not in the base class for either V1 or V2. It is in the IBM classes for both V1 and V2. I think the IBM provider is the only one that implements qiskit pulse support, so it is hard to distinguish between what is IBM-specific and what is pulse-specific. I think it is fair for experiments referencing the drive frequencies to use pulse-specific API's (and so probably fair to use `defaults()` unless all of its utility is supposed to be moved to `Target`).

I don't think it is worth giving up access to the measurement frequencies when using V2 because the measurement frequencies are still available in the same place we have been referencing for V1.

Moving away from `defaults()` seems fine if that is the direction that we are supposed to go. It might be better to do that in a separate step from initial V2 compatibility. This is a little pedantic but I think the frequencies should live in two places. One place (like `backend.properties()`) can hold resonant frequencies of the device and the other (like `backend.defaults()` or maybe `backend.target` in the future) can hold frequencies used by the control electronics. For IBM superconducting transmon qubits, readout is done by coupling to a resonator whose frequency changes (by 2 chi) depending on the state of the qubit. Typically readout is done by driving half-way in between these two frequencies. There is less distinction for qubits but even there for finite ZZ one needs to decide to drive at the frequency of the qubit with its neighbors in 0 or with them in 0+1.

Reply:
One reason my last comment is pedantic -- while in calibrations we are interested in determining the frequencies at which to drive the control and measurement tones, all of the usages of drive_freqs/meas_freqs currently are rough numbers -- used either to set default frequencies for calibrations and center frequencies for spectroscopy. The resonant frequencies are probably good enough as starting points.